client/android/CMakeLists.txt

```diff
@@ -1,17 +1,20 @@
-#需要的最cmake版本
+# version
 cmake_minimum_required(VERSION 3.4.1)
 
-#添加预编译宏定义参数，此次的作用是开启配置文件的引入!
+# We are build on android platform, so we need add def "ANDROID"
+# NDK version for SDK 19 doesn't implement the whole C++11 standard in the STL.
+# see: https://stackoverflow.com/questions/44736135/ndk-clang-error-undefined-reference-to-localeconv
+# so we need add def getlocaledecpoint()='.'
 add_definitions(-DANDROID -D"getlocaledecpoint()='.'" -DZ_SOLO)
```

Comment:
I've one question.
Is the line working also if you add "\" to it, like this ?
`add_definitions(-DANDROID -D"getlocaledecpoint\(\)='.'" -DZ_SOLO)`

Because currently I cannot execute the cmake on my linux (for tests) with `getlocaledecpoint()=` but it works with `getlocaledecpoint\(\)`